src/Chainweb/Pact/Backend/Types.hs

```diff
@@ -224,14 +226,18 @@ data BlockState = BlockState
     , _bsModuleNameFix :: Bool
     , _bsSortedKeys :: Bool
     , _bsLowerCaseTables :: Bool
+    , _bsModuleCache :: DbCache PersistModuleData
     }
-    deriving Show
 
 emptySQLitePendingData :: SQLitePendingData
 emptySQLitePendingData = SQLitePendingData mempty mempty mempty mempty
 
-initBlockState :: BlockHeight -> BlockState
-initBlockState initialBlockHeight = BlockState
+initBlockState
+    :: Int
+        -- ^ Module Cache Limit (in bytes of corresponding rowdata)
```

Comment:
Hmm should we newtype `Natural` as `ModuleCacheLimitBytes` or something since it shows up here needing a comment and in config, document that it's bytes, etc?